packages/zoe/src/zoeService/zoe.js

```diff
@@ -168,6 +170,25 @@ const makeDurableZoeKit = ({
     });
   };
 
+  const ZoeConfigI = M.interface('ZoeConfigFacet', {
+    updateZcfBundleId: M.call(M.string()).returns(),
+  });
+
+  const zoeConfigFacet = prepareExo(zoeBaggage, 'ZoeConfigFacet', ZoeConfigI, {
+    updateZcfBundleId(bundleId) {
+      E.when(
+        getZcfBundleCap({ id: bundleId }, vatAdminSvc),
+        bundleCap => {
+          zcfBundleCap = bundleCap;
+        },
+        e => {
+          console.error(`'🚨 unable to update ZCF Bundle: `, e);
```

Comment:
I see

![image](https://github.com/Agoric/agoric-sdk/assets/273868/af0dbf93-0b69-4d46-a12d-c434c2558891)


What's the weird character within the literal string after the open single quote? Why is there no close single quote?